### PR TITLE
Notification for node version update

### PR DESCRIPTION
**Pull request by MykolasMaciekus**
- API call to check current release version in GitHub
- Refactored notification display logic
- Required styling for update notification
- Remote storage utilization to check last check time to avoid calling api.github.com on every rerender

### CODE REVIEW

File: src/api/node-version.management.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2022 BlockDev AG
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const checkNodeVersion = async () => {
```
Comment (mdomasevicius) [inline src/api/node-version.management.ts:8]: You are not checking node version in `checkNodeVersion` you are fetching latest node version.

File: src/api/node-version.management.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      method: 'GET',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    })
+
+    const currentRelease = await currentReleaseResponse.json()
+    return currentRelease.tag_name
+  } catch (err) {
+    return err
```
Comment (mdomasevicius) [inline src/api/node-version.management.ts:20]: Let's not have double return types. Typically if you want to pass down errors you should put re-throw them down the stack by: `throw err`

File: src/Pages/Authenticated/Components/Notifications/Notifications.tsx [hunk 3]
```diff
@@ -81,24 +83,13 @@ export const Notifications = () => {
   }, [])
 
   const lastCheck = useAppSelector(remoteStorage.selector<number>(KEY_CURRENT_NODE_VERSION_LAST_CHECK))
-  console.log(useAppSelector(remoteStorage.selector<number>(KEY_CURRENT_NODE_VERSION_LAST_CHECK)))
   useEffect(() => {
     ;(async () => {
       try {
-        if (!lastCheck || now > lastCheck + 2000) {
+        if (!lastCheck || now > lastCheck + 7200000) {
```
Comment (mdomasevicius) [inline src/Pages/Authenticated/Components/Notifications/Notifications.tsx:89]: in the case that humans are terrible at translating milliseconds I'd say move this magic number to constant like `const TWO_HOURS_MS = 2 * 60 * 60 * 1000`

File: src/theme/light.ts [hunk 4]
```diff
@@ -136,6 +136,9 @@ const light: Omit<Theme, 'common'> = {
       neutral: {
         background: themeCommon.colorGrayBlue + alphaToHex(0.2),
       },
+      update: {
+        background: themeCommon.colorYellow + alphaToHex(0.1),
```
Comment (mdomasevicius) [inline src/theme/light.ts:140]: I wonder if we could use this so we could just write something like

```
background: themeCommon.colorYellow + 0.1.hex(),
```

File: src/Pages/Authenticated/Components/Notifications/List.tsx [hunk 5]
```diff
@@ -22,25 +24,20 @@ const Container = styled.div`
...
-  )
+interface Props {
+  list: Notification[]
+}
+export const List = ({ list }: Props) => {
+  const Notifications = useMemo(() => {
+    return list.map((notification) => {
+      return (
+        <Card
+          variant={notification.variant as CardVariant}
```
Comment (mdomasevicius) [inline src/Pages/Authenticated/Components/Notifications/List.tsx:35]: `as` cast seems not necessary

**Review by mdomasevicius — CHANGES_REQUESTED** [review]

File: src/theme/light.ts (hunk 4 above)
Comment (MykolasMaciekus) [reply src/theme/light.ts:140]: Looks a lot like python to me, have you considered how can we implement this?

**Review by mdomasevicius — APPROVED** [review]

**Pull request merged**